wiki/Difficulties/en.md

```diff
@@ -1,141 +1,59 @@
-Difficulty appropriate gameplay elements
-==========================================
+# Difficulties
 
-Star Rating and Web Icons
--------------------------
+*Not to be confused with: [Beatmaps](/wiki/Beatmaps).*
 
-![Star Rating Range](SR_range.png "Star Rating Range")
+*See also: [Ranking Criteria § Mapset](/wiki/Ranking_Criteria/#mapset).*
 
--   From the [Ranking Criteria](/wiki/Ranking_Criteria):
-    -   Below 1.5: Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
-    -   Below 2.25: Normal ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon")
-    -   Below 3.75: Hard ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")
-    -   Below 5.25: Insane ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")
-    -   Above 5.25: Expert ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
+The difficulty of a beatmap *describes* the level of skill needed to complete a beatmap.
 
-This is the range that decides which web icons are used depending on the star rating of any given map. Following the icon range is a good estimate to achieve a balanced difficulty spread. There are some guidelines you can follow to improve the difficulty spread of your mapset.
+## Star rating
 
-osu! difficulty spreads
---------------------------------------------------
+![Star rating ranges](SR_range.png "Star rating ranges")
 
-From the [Ranking Criteria](/wiki/Ranking_Criteria):
+From the [Ranking Criteria](/wiki/Ranking_Criteria), a star rating of:
 
-"**The difficulties in the mapset must be in a consecutive order. Easy or Normal can be skipped if the gap in the star rating spread allows it. If your mapset has two difficulties, one of them cannot be an Insane or Expert. The lowest difficulty must be below 2.0 stars.**"
+- 1.5 (or below) is Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
```

Comment:
1.5★ (or below) is called Easy, maybe?
not sure about this one, same for the ones below?